src/confluent_kafka/schema_registry/avro.py

```diff
@@ -67,6 +67,24 @@ def _schema_loads(schema_str):
     return Schema(schema_str, schema_type='AVRO')
 
 
+def _resolve_named_schema(schema, schema_registry_client, named_schemas=None):
+    """
+    Resolves named schemas referenced by the provided schema recursively.
+    :param schema: Schema to resolve named schemas for.
+    :param schema_registry_client: SchemaRegistryClient to use for retrieval.
+    :param named_schemas: Dict of named schemas resolved recursively.
+    :return: named_schemas dict.
+    """
+    if named_schemas is None:
+        named_schemas = {}
+    for ref in schema.references:
+        referenced_schema = schema_registry_client.get_version(ref.subject, ref.version)
```

Comment:
Can we do a batch call to schema registry?

Reply:
I checked on this, it doesn't looks like it's available directly through schema registry. https://docs.confluent.io/platform/current/schema-registry/develop/api.html#post--subjects-(string-%20subject)-versions